bigcommerce/customer_login_token.py

```diff
@@ -0,0 +1,55 @@
+import os
+import time
+import uuid
+import jwt
+
+# Get the client_secret needed to sign tokens from the environment
+# Intended to play nice with the Python Hello World sample app
+# https://github.com/bigcommerce/hello-world-app-python-flask
+client_secret = os.getenv('APP_CLIENT_SECRET', None)
+
+
+class CustomerLoginTokens:
+    @classmethod
+    def create(cls, client, id, redirect_url=None, request_ip=None):
+
+        if not client_secret:
+            raise AttributeError('No OAuth client secret specified in the environment, '
+                                 'please specify an APP_CLIENT_SECRET')
+
+        try:
+            client_id = client.connection.client_id
+        except AttributeError:
+            raise AttributeError('No client ID found in the connection - '
```

Comment:
maybe specify how to configure the OAuth API connection in the error message? 

e.g. `Please ensure you have done Foo.Bar` etc.